ISPN-12657 Cross-Site automatic state transfer (#9202)
infinispan/infinispan: cli/src/main/resources/help/site.adoc

*--site*='SITE_NAME'::
Specifies a backup location.

STATE TRANSFER MODE OPTIONS
---------------------------

*--mode*='MODE'::

> Sets the state transfer mode (`MANUAL` or `AUTO`)

Reviewer: ```suggestion
Sets the state transfer mode. Values are `MANUAL` or `AUTO`.
```

Reviewer: @pruivo Is `MANUAL` the default?

Reviewer: Nevermind. I found this in the schema.

`site clear-push-state-status --cache=myCache` +
Clears the status of the push state operation for `mycache`.

`site state-transfer-mode --cache=myCache --site=NYC` +

> Retrieves the state transfer mode for the cache `mycache` and remote backup `NYC`

Reviewer: ```suggestion
Retrieves the state transfer mode for `mycache` to `NYC`.
```

`site state-transfer-mode --cache=myCache --site=NYC --mode=AUTO` +

> Sets the state transfer mode to `AUTO` for the cache `mycache` and remote backup `NYC`

Reviewer 2: It seems strange to configure the value by just setting a `--mode=AUTO` flag. Shouldn't we have dedicated commands for get & set? e.g.

```
site state-transfer-mode get --cache=myCache --site=NYC
site state-transfer-mode set --cache=myCache --site=NYC AUTO
```

Author: ok
